symengine/codegen.h

```diff
@@ -109,6 +109,70 @@ class CodePrinter : public BaseVisitor<CodePrinter, StrPrinter>
             o << "pow(" << apply(a) << ", " << apply(b) << ")";
         }
     }
+    void bvisit(const Abs &x)
+    {
+        std::ostringstream s;
+        s << "fabs(" << apply(x.get_arg()) << ")";
+        str_ = s.str();
+    }
+    void bvisit(const Ceiling &x)
+    {
+        std::ostringstream s;
+        s << "ceil(" << apply(x.get_arg()) << ")";
+        str_ = s.str();
+    }
+    void bvisit(const Gamma &x)
+    {
+        std::ostringstream s;
+        s << "tgamma(" << apply(x.get_arg()) << ")";
+        str_ = s.str();
+    }
+    void bvisit(const LogGamma &x)
+    {
+        std::ostringstream s;
+        s << "lgamma(" << apply(x.get_arg()) << ")";
+        str_ = s.str();
+    }
+    void bvisit(const Max &x)
+    {
+        std::ostringstream s;
+        const auto& args = x.get_args();
+        switch (args.size()) {
+        case 0:
+            throw SymEngineException("Impossible");
+        case 1:
+            s << apply(args[0]); break;
```

Comment:
This is impossible as well.

Reply:
Yes, I know, just thought the printer could have the option in case of Min/Max changing but thinking more about it I cannot see why that would ever change. Will let case 0 & case 1 be the same.